### PR TITLE
Adobe Creative Cloud Libraries (Certified Connector)

### PR DESCRIPTION
**Pull request by dches**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

### CODE REVIEW

File: certified-connectors/Adobe Creative Cloud Libraries/README.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Adobe Creative Cloud Libraries
+Adobe Creative Cloud Libraries lets you connect Creative Cloud Libraries with thousands of the most popular apps so you can automate your work and have more time for what matters most-no code required.
```
Comment (sriyen-msft) [inline certified-connectors/Adobe Creative Cloud Libraries/README.md:2]: @dches Your connector readme.md file is incomplete. Please follow all the instructions mentioned in the [documentation](https://github.com/microsoft/PowerPlatformConnectors#readmemd) and [here](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/readme.md) is the template for readme.md.

File: certified-connectors/Adobe Creative Cloud Libraries/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,616 @@
...
+                  },
+                  "description": "data"
+                },
+                "config": {
+                  "type": "object",
+                  "properties": {
+                    "url": {
+                      "type": "string",
+                      "description": "url",
+                      "title": "",
```
Comment (sriyen-msft) [inline certified-connectors/Adobe Creative Cloud Libraries/apiDefinition.swagger.json:464]: @dches Empty property values can be removed until unless you have a specific reason to keep them.

File: certified-connectors/Adobe Creative Cloud Libraries/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,616 @@
...
+      }
+    }
+  },
+  "definitions": {},
+  "parameters": {
+    "clientId_OAuthWeb": {
+      "in": "header",
+      "name": "x-api-key",
+      "type": "string",
+      "default": "0fa67f2c25804955b2c7b41d77db612e",
```
Comment (sriyen-msft) [inline certified-connectors/Adobe Creative Cloud Libraries/apiDefinition.swagger.json:561]: @dches Please remove the secret-look-a-like values from the swagger due to Microsoft security policies.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/Adobe Creative Cloud Libraries/apiDefinition.swagger.json (hunk 3 above)
Comment (dches) [reply certified-connectors/Adobe Creative Cloud Libraries/apiDefinition.swagger.json]: Replaced this value with a placeholder

File: certified-connectors/Adobe Creative Cloud Libraries/apiDefinition.swagger.json (hunk 2 above)
Comment (dches) [reply certified-connectors/Adobe Creative Cloud Libraries/apiDefinition.swagger.json]: Added title for this property

File: certified-connectors/Adobe Creative Cloud Libraries/README.md (hunk 1 above)
Comment (dches) [reply certified-connectors/Adobe Creative Cloud Libraries/README.md:2]: updated readme.md file

File: certified-connectors/Adobe Creative Cloud Libraries/README.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
```
Comment (sriyen-msft) [inline certified-connectors/Adobe Creative Cloud Libraries/README.md:3]: @dches Publisher section is missing, please add it.

File: certified-connectors/Adobe Creative Cloud Libraries/apiDefinition.swagger.json [hunk 5]
```diff
@@ -0,0 +1,616 @@
...
+      }
+    }
+  },
+  "definitions": {},
+  "parameters": {
+    "clientId_OAuthWeb": {
+      "in": "header",
+      "name": "x-api-key",
+      "type": "string",
+      "default": "{{ client id }}",
```
Comment (sriyen-msft) [inline certified-connectors/Adobe Creative Cloud Libraries/apiDefinition.swagger.json:561]: @dches Can you please let us know the reason to pass client id in the header of the actions?

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/Adobe Creative Cloud Libraries/README.md (hunk 4 above)
Comment (dches) [reply certified-connectors/Adobe Creative Cloud Libraries/README.md:3]: @sriyen-msft As I see in the description [here](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/readme.md) - Publisher is required only for Independent-Publisher-Connector. Should I add this section for "certified-connector"?

File: certified-connectors/Adobe Creative Cloud Libraries/apiDefinition.swagger.json (hunk 5 above)
Comment (dches) [reply certified-connectors/Adobe Creative Cloud Libraries/apiDefinition.swagger.json:561]: @sriyen-msft Adobe API requires this header for authorization.

Comment (sriyen-msft) [reply certified-connectors/Adobe Creative Cloud Libraries/apiDefinition.swagger.json:561]: @dches It is not a good idea to have secure information in the swagger. Can you please start an email thread with us(ConnectorPartnerMgmtTeam@service.microsoft.com) to understand the risks to have a client id in the header of all requests that are defined in the swagger? And we/our engineering team may provide you if we have a resolution.

File: certified-connectors/Adobe Creative Cloud Libraries/README.md (hunk 4 above)
Comment (dches) [reply certified-connectors/Adobe Creative Cloud Libraries/README.md:3]: added publisher to the readme.md file

File: certified-connectors/Adobe Creative Cloud Libraries/apiDefinition.swagger.json (hunk 5 above)
Comment (dches) [reply certified-connectors/Adobe Creative Cloud Libraries/apiDefinition.swagger.json:561]: @sriyen-msft Ok, I will talk about this header with my team.

Comment (dches) [reply certified-connectors/Adobe Creative Cloud Libraries/apiDefinition.swagger.json]: @sriyen-msft  We removed this parameter from the swagger file.

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**